### ISSUE TITLE
[Bug]: <Cant Login into account after password resets>

### ISSUE THREAD

**Issue by Blindsinister** [MelvorIdle/melvoridle.github.io#4672]
### Describe the bug

First bug report here, My account user is Blind, I’ve reset my account numerous times tonight and in the past trying to get onto my account and never having any luck, I’ve done the password resets to my email and changed my password, then within seconds attempting to login with then new password, and never having luck getting into my account, I thought I’d give this bug report a shot because I would say I’ve tried atleast 30 times with password resets in the last year or so. I’m thinking my account is bugged. Thanks in advance for helping me understand what has happened.

### Expected behaviour

I just expect to get into my account after changing my password, I think my account is bugged.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Steam
- [ ] Epic Games
- [X] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which expansions are purchased and active at the time of the bug?

- [ ] Throne of the Herald
- [ ] Atlas of Discovery
- [ ] Into the Abyss

### Which version of the game are you experiencing this bug on?

v1.3

### Game Subversion

?11182

### Are you using any mods?

No

**Comment by Blindsinister** [comment]
Any help?

**Comment by MelvorIdle** [comment]
Are you attempting to login using the username and not the email? The login form only accepts username

**Comment by Blindsinister** [comment]
Yes I’ve been logging in with my username, I think my account is bugged or something, Uhm I’ve taken screenshots of each step I did, changed the password and still tells me Incorrect username or password, I’m signing in with my account under the name Blind

**Comment by Flurryous** [comment]
Really late so dunno if you need help anymore but if the username is Blind (and not the character file name), then maybe it's two i's with the first one being capitalized?
